lib/LocalVerifiableCredentialStore.js

```diff
@@ -0,0 +1,137 @@
+/*!
+ * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+
+/**
+ * Each instance of this API is associated with a single edv and performs
+ * initialization (ensures required indexes are created).
+ */
+import PouchDB from 'pouchdb';
+import pouchFind from 'pouchdb-find';
+PouchDB.plugin(pouchFind);
+
+export default class LocalVerifiableCredentialStore {
+  constructor({db, dbName, profileId}) {
+    this.db = db || new PouchDB(dbName);
+    this.profileId = profileId;
+    this.db.createIndex({
+      index: {
+        fields: [
+          'content.id',
+          'meta.parentId',
+          'meta.profileId',
+          'meta.displayable'
+        ]
+      }
+    });
+  }
+
+  /**
+   * Gets a verifiable credential by its ID.
+   *
+   * @param {string} id
+   */
+  async get({id}) {
+    const {docs: [doc]} = await this.db.find({
+      selector: {
+        'meta.profileId': this.profileId,
+        'content.id': id
+      },
+      include_docs: true,
+      attachments: true
+    });
+    if(!doc) {
+      const err = new Error('Verifiable Credential not found.');
+      err.name = 'NotFoundError';
+      throw err;
+    }
+    return doc;
+  }
+
+  /**
+   * Gets all verifiable credential instances that match the given parameters.
+   *
+   * @param {boolean} displayable
+   * @param {string} parentId
+   *
+   * @return {Promise<Array>} List of matching VCs
+   */
+  async find({displayable, parentId}) {
+    const selector = {
+      'meta.profileId': this.profileId,
+    };
+    if(parentId) {
+      selector['meta.parentId'] = parentId;
+    }
+    if(displayable) {
+      selector['meta.displayable'] = displayable;
+    }
+    const {docs} = await this.db.find({
+      selector,
+      include_docs: true,
+      attachments: true
+    });
+    return docs.map(({content, meta}) => {
+      return {
+        content,
+        meta
+      };
+    });
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#advanced_syntax

I assume this is stripping out some unwanted metadata, if that's the case, a comment would be helpful.  If it's not the case, is the map needed?

```suggestion
    return docs.map(({content, meta}) => ({content, meta}));
```